### PR TITLE
LPD-41067 Change the reference from the Collection Display to the Object from ID to ClassName

### PR DESCRIPTION
**Pull request by icgsbr**
Coming from --> https://github.com/liferay-page-management/liferay-portal/pull/16565

### CODE REVIEW

**Comment by liferay-page-management** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by icgsbr** [comment]
ci:test:sf

File: modules/apps/site-initializer/site-initializer-extender/site-initializer-extender-test-bundle-1/src/main/resources/site-initializer/layouts/1_test-objects-layout/page-definition.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
 				"definition": {
 					"collectionConfig": {
 						"collectionReference": {
-							"className": "com.liferay.object.internal.info.collection.provider.ObjectEntrySingleFormVariationInfoCollectionProvider_C_[$OBJECT_DEFINITION_ID:TestObjectDefinition3$]"
+							"className": "[$OBJECT_DEFINITION_ID:TestObjectDefinition3$]"
```
Comment (carolmariaabb) [inline modules/apps/site-initializer/site-initializer-extender/site-initializer-extender-test-bundle-1/src/main/resources/site-initializer/layouts/1_test-objects-layout/page-definition.json:8]: @icgsbr shouldn't you use `OBJECT_DEFINITION_CLASS_NAME` instead?

**Review by carolmariaabb — CHANGES_REQUESTED** [review]
@icgsbr It seems that these changes need to be made in two other places.

See [here](https://github.com/carolmariaabb/liferay-portal/blob/a1ff55b871298797bc8ec371b4b2046c001d1a42/workspaces/liferay-tryitnow-workspace/client-extensions/liferay-tryitnow-site-initializer/site-initializer/layouts/5_view-tickets/page-definition.json#L522) and [here](https://github.com/carolmariaabb/liferay-portal/blob/a1ff55b871298797bc8ec371b4b2046c001d1a42/workspaces/liferay-tryitnow-workspace/client-extensions/liferay-tryitnow-site-initializer/site-initializer/segments-experiences/view-tickets/admin-view/page-definition.json#L568).

**Comment by icgsbr** [comment]
ci:test:sf

**Review by lfbesada — APPROVED** [review]
@icgsbr Thank you very much! :clap:

**Comment by lfbesada** [comment]
@carolmariaabb I think everything is OK now.
Could you please take a look?
Thank you very much :heart:

**Review by carolmariaabb — APPROVED** [review]

**Comment by carolmariaabb** [comment]
ci:test:relevant

**Comment by lfbesada** [comment]
ci:report:66577305

**Comment by lfbesada** [comment]
ci:forward:force

**Pull request closed without merging**